README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Athens Banner](./docs/static/banner.png)
 
-[![Build Status](https://cloud.drone.io/api/badges/gomods/athens/status.svg)](https://cloud.drone.io/gomods/athens)
+[![Build Status](https://github.com/gomods/athens/actions/workflows/ci.yml/badge.svg)](https://github.com/gomods/athens/actions)
```

Comment:
Sorry, I just realised that the previous url wasn't pointing to the `ci.yml` in the repository 🤦 and was actually the correct page. Can we revert back and also include the branch name filter: https://github.com/gomods/athens/actions/workflows/ci.yml?query=branch%3Amain. The badge url considers the default branch so that shouldn't need the branch name filter.

Apologies again for the misunderstanding. 🙇